grapheme_test.go

```diff
@@ -14,11 +14,13 @@ func ExampleGraphemes() {
 	// Output: [1f44d 1f3fc] [21]
 }
 
-// The test cases for the simple test function.
-var testCases = []struct {
+type testcase = struct {
```

Comment:
Can you please name this `testCase` (capital "C")? I use camel case and "test case" are two words.

Reply:
Done.